reactfire/firebaseApp/index.tsx

```diff
@@ -6,7 +6,7 @@ export * from './sdk';
 type FirebaseAppContextValue = firebase.app.App;
 
 // INVESTIGATE I don't like magic strings, can we have export this in js-sdk?
-export const DEFAULT_APP_NAME = '[DEFAULT]';
+const DEFAULT_APP_NAME = '[DEFAULT]';
```

Comment:
looks like this might not be needed any more?

Reply:
I'm still using within this module (`FirebaseAppProvider`) no longer using in `sdk.ts`, thus dropping the export.

Reply:
👍